### ISSUE TITLE
Compare your results

### ISSUE THREAD

**Issue by name333** [lyndonzheng/Synthetic2Realistic#19]
Hello, I ran the model you put on Google.My results are darker than yours.I found the image max_depth 255 of your model test, but the maximum value of my training model test is 80. Of course, your image is evaluated correctly. I am thinking about the VKITTI_GT clipped to 80. We are testing the result should be 0-80 is correct? How did you become 0-255 during training? Or can you tell me how VKITTI_GT is clipped to 80.
The picture on the right is your result, the picture on the left is mine.
![image](https://user-images.githubusercontent.com/35323680/50681446-7d73b180-1046-11e9-9649-cf776a559a43.png)
Some results have been sent to your mailbox. If you have time, I hope you can take a look.

Thank you.

**Comment by lyndonzheng** [comment]
@name333 Since the value range of RGB is 0-255, I normalize the ground truth depth value 0-80 to 0-255 during the training. If the max_depth value in your results is 80, you can change the value (255) of function "load_depth" in util/data_kitti.py. You need to make sure you test image and ground truth hold same depth scale.

**Comment by name333** [comment]
Thanks for your reply, I think I understand where the previous problem is. I think it is more correct to normalize the vkitti depth GT to (0-255) for training. The first two days I tried normalization to (0-255), maybe I was clipped to (0-80), and then normalized to (0-255) the wrong way. Can you provide this part of the code? I want to be consistent with your process. I don't know if my request is appropriate, but I still want to ask you to consider it. Help me achieve it better. Thanks!

**Comment by yipengm** [comment]
Hello, I also used the model you give on the google and ran it on the test.py. I found out the output is the same as the prediction in "Outdoor_wsupervised/predicted_depth_vk" and the evaluation result is close to the value on the paper. (Something like this)
![0000000000_fake_depth](https://user-images.githubusercontent.com/25646118/50741727-5399c680-11cf-11e9-97e4-804990d836cd.png)

However, I found that you also provide predictions in "Outdoor_wsupervised/predicted_depth_k", which performs better than that in "Outdoor_wsupervised/predicted_depth_vk".(Something like this)

![0000000000_fake_depth](https://user-images.githubusercontent.com/25646118/50741730-62807900-11cf-11e9-8782-9e7d0241a916.png)

How can we get that prediction? Thank you very much.

**Comment by lyndonzheng** [comment]
@name333 I just use the code of [monodepth] (https://github.com/mrharicot/monodepth/blob/master/utils/evaluate_kitti.py) to get the dense depth maps as ground truth, and normalize the depth range to [0,255]. The example ground truth is that: ![0000000000](https://user-images.githubusercontent.com/8929977/50743793-03664700-1257-11e9-85ce-435f16c1602b.jpg)

**Comment by lyndonzheng** [comment]
@yipengm The predicted_depth_k has no depth value in the top part. Those predicted depth maps come from the supervised method. It's a baseline of supervised training. If you want to get the similar results, you can directly train the model in supervised way. However, the supervised depth prediction is not the key problem we try to solve in this project. There are many other good works to design new structure and loss function for the [supervised depth prediction](http://www.cvlibs.net/datasets/kitti/eval_depth.php?benchmark=depth_prediction).

**Comment by name333** [comment]
@lyndonzheng Yes, it is the way to get KITTI GT, can you tell me how to deal with vkitti gt clipped to 80? I want to retrain. Thank you

**Comment by lyndonzheng** [comment]
@name333 The clipped processing can be found in [issues 22](https://github.com/lyndonzheng/Synthetic2Realistic/issues/22)
